dashboard/src/dashboard.py

```diff
@@ -436,6 +441,17 @@ def read_job_statuses(ci_jobs, ci_jobs_table_enabled, liveness_table_enabled):
         return None
 
 
+def production_smoketests_status(ci_jobs):
+    """Read total number of remembered builds and failed builds as well."""
+    job_url = ci_jobs.get_job_url("production", "smoketests")
+    api_query = jenkins_api_query_build_statuses(job_url)
+    response = requests.get(api_query)
+    builds = response.json()["builds"]
+    total_builds = [b for b in builds if b["result"] is not None]
+    failed_builds = [b for b in builds if b["result"] == "SUCCESS"]
```

Comment:
you mean `result != "SUCCESS"` ?

Reply:
actually not, I meant success_builds = .... oh, very nice catch! Thank you